.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "develop"
+    ]
+}
```

Comment:
.gitignore should prevent this file from being included. I don't know what happened here.

Please remove this file.

Reply:
![image](https://github.com/user-attachments/assets/658ee9f9-e1c5-4568-bdd3-707601cca427)

it was ignored - not sure either

Reply:
either way, I removed it.